pyslackers_website/blog/templates/blog/post_detail.html

```diff
@@ -21,7 +21,7 @@ <h4 class="ui horizontal divider header" style="color: darkgrey;">
       <a class="ui label" href="{% url 'blog:index' %}?tag={{ tag }}">{{ tag.name | title }}</a>
     {% endfor %}
     
-    <div class="ui container" style="margin-top: 15px;">
+    <div class="ui container" style="margin-top: 15px 0 60px;">
```

Comment:
I believe for this to work properly we want `margin-top` -> `margin`, otherwise

Reply:
ah, missed it... that's what happens when you test in dev tools